src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import './footer.css';
+import logo from '../../static/images/logo.png';
+
+import facebook from '../../static/icons/facebook.svg';
+import youtube from '../../static/icons/youtube.svg';
+import github from '../../static/icons/github.svg';
+import twitter from '../../static/icons/twitter.svg';
+import google from '../../static/icons/google.svg';
```

Comment:
import này có vẻ dài dòng quá, rút ngắn bằng cách [tạo 1 file index rồi re-export lại ](https://stackoverflow.com/questions/34072598/es6-exporting-importing-in-index-file)